src/asm/section.c

```diff
@@ -1016,3 +1016,25 @@ void sect_PopSection(void)
 	sectionStack = entry->next;
 	free(entry);
 }
+
+bool sect_IsSizeKnown(struct Section const *sect)
+{
+	if (!sect)
```

Comment:
I think the function should be "nonnull" instead; this would be both consistent with the rest of the codebase, and coherent with most use cases.

Reply:
Nothing actually uses `attr_(nonnull)` yet in `master`, but may as well start now.